### DISCUSSION TITLE
[NPM] Update

### DISCUSSION THREAD

**Discussion by thytetgc** [tteck/Proxmox#2340] [Help Needed] [answered]
What is the best way to update NPM LXC?

**Comment by tteck** [comment]
Follow the update instructions.

**Reply by tteck** [reply, marked as answer]
If the instance was created after October 5, 2023, simply type `update` in the LXC console. However, if it was created on or before that date, please execute the full command in the LXC console.

**Reply by liveware-problems** [reply]
Probably a silly question, but can running the full command then create the `update`  command script in an older LXC? Much easier to just run `update` rather than finding the full command for the LXC creation...

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/1900#discussioncomment-7226935

**Reply by liveware-problems** [reply]
hah thanks for the swift reply @tteck ! I just looked at a couple of the newer containers I've spun up using your scripts and saw the file in `/usr/bin` with the call to the full command 👍
